src/route/routes.js

```diff
@@ -5,15 +5,16 @@ export const routes = {
     },
     search: {
         routerPath: '/search',
-        getPathWithParams: (id = '') => `/search?id=${id}`
+        getPathWithParams: (query = '') =>
```

Comment:
I guess we dont need test since you gave up on passing the input value in url?

Reply:
yup, gonna remove unused code